_posts/2019-06-19-snapcraft-summit-montreal.md

```diff
@@ -1,7 +1,9 @@
 ---
 title: Snapcraft Summit Montréal
-summary: Updates in Crystal distributions and other news
+description: Updates in Crystal distributions and other news
```

Comment:
Is this change OK? I don't think `description` is used. Most post have`summary`, except for a few (and that's probably an error).

Reply:
@straight-shoota ping

Reply:
And the one below 👇